boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp_0_8_0.overlay

```diff
@@ -0,0 +1,7 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <nrf54h20dk_nrf54h20_0_8_0.dtsi>
```

Comment:
Does this work for you? For me, this resulted in a build error, whereas
```suggestion
#include "nrf54h20dk_nrf54h20_0_8_0.dtsi"
```
works